Example/ViewController+FreeDrive.swift

```diff
@@ -153,6 +153,7 @@ extension ViewController {
     }
     
     func edgeNames(identifier: RoadGraph.Edge.Identifier) -> [String] {
+        if navigationMapView == nil { return [] }
```

Comment:
Fixes a rare crash when `navigationMapView` can be unavailable right after starting active navigation.